File: TestProject2/UnitTest1.cs
// Created: 2020-08-08 16:33
using Microsoft.VisualStudio.TestTools.UnitTesting;
using static System.Net.Mime.MediaTypeNames; // Assuming this is the actual namespace of your main project
using System.IO;
using System;
using SCO203;

namespace TestProject2
{
    [TestClass]
    public class UnitTest1
    {
        [TestMethod]
        public void TestLoginWithValidCredentials()
        {
            string username = "admin";
            string password = "admin";
            string expectedMessage = "Login Successful";

            Form1.Login_Click(username, password);

            Assert.AreEqual(expectedMessage, "Login Successful");


        }
    }
}

File: TestProject5/UnitTest1.vb
Imports Microsoft.VisualStudio.TestTools.UnitTesting
Imports System.Net.Mime.MediaTypeNames
Imports System.IO
Imports System
Imports grp203

<TestClass()>
Public Class UnitTest1
    <TestMethod()>
    Public Sub TestLoginWithValidCredentials()
        Dim username As String = "admin"
        Dim password As String = "admin"
        Dim expectedMessage As String = "Login Successful"

        Dim form As New Form1() ' Using Form1 from the imported grp203 namespace

        form.Login_Click(username, password)

        Assert.AreEqual(expectedMessage, "Login Successful")
    End Sub
End Class
